### DISCUSSION TITLE
Indented line blocks do not render indented

### DISCUSSION THREAD

**Discussion by mwchase** [pradyunsg/furo#285] [Potential Issue] [answered]
I was formatting some poetry using https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#line-blocks, and the HTML came out correctly, but Furo didn't indent the indented lines. The key appears to be styling `.line-block .line-block`. The relevant lines in the sphinx-rtd theme appear to be
```
.rst-content .line-block {
 margin-left:0;
 margin-bottom:24px;
 line-height:24px
}
.rst-content .line-block .line-block {
 margin-left:24px;
 margin-bottom:0
}
```
For now, I'm working around this using the following lines of custom CSS, cribbed from another theme:
```
.line-block .line-block {
  margin-left: 1.5em;
}
```

The source material I tried this with was something like
```
| Here is a line
|   Here is an indented line
| Here is a non-indented line
```
The middle line is in a pair of line-block elements, which Furo does not seem to style.

**Comment by pradyunsg** [comment, marked as answer]
Hiya! Could you please file a [trackable task issue](https://github.com/pradyunsg/furo/issues/new?assignees=&labels=&template=issue.yml) for this? This certainly seems like something worth doing!
